server/src/main/java/io/spine/server/entity/storage/ColumnRecords.java

```diff
@@ -36,7 +36,7 @@
  */
 public final class ColumnRecords {
 
-    /** Prevent initialization of the utility class */
+    /** Prevents initialization of the utility class. */
```

Comment:
It's “instantiation” (i.e. we cannot create instances of this class), not “initialization”. The class _is_ initialized.

Please also look for such errors around.